### ISSUE TITLE
Document how to create a client

### ISSUE THREAD

**Issue by koalp** [elabftw/elabapi-python#8]
Hello, 

I’m trying to use the v2 python library but am a bit lost in the documentation.

I found out that the following works:
```python3
config = elab.Configuration()
config.host = HOST
config.api_key = {"Authorization": API_TOKEN}
```
But in the examples, you set `api_key`, `api_key_prefix` and then a workaround with `set_default_header`. What is the recommended way to create a client ?
Will there be a configuration that can be instantiated in an easier way ? (`Configuration(host=HOST, api_token=API_TOKEN)` for example)

**Comment by NicolasCARPi** [comment]
Hello fellow koala :koala: ,

> and then a workaround 

I'll have to check if the workaround is still necessary.

> What is the recommended way to create a client ?

As shown in the examples.

> Will there be a configuration that can be instantiated in an easier way ?

I didn't try but the keyword approach you mention might work AFAIK.
I'm not in control of how the library works, because it is generated by swagger-codegen.

**Comment by koalp** [comment]
>  I'll have to check if the workaround is still necessary.

[The issue](https://github.com/swagger-api/swagger-codegen/issues/10060) doesn’t seem to be closed =/ (if it is the good one)

> I didn't try but the keyword approach you mention might work AFAIK.
I'm not in control of how the library works, because it is generated by swagger-codegen.

When I tried it didn’t work (the to generated `__init__` doesn’t seem to take arguments). I looked if could be added in the library with swagger-codegen but didn’t found a proper way. A patch could be applied to add features to modify the `Configuration` class in `configuration.py` but that may be brittle and doesn’t solve the workaround issue =/

I think that the issue can be closed, unless you want to track workaround issue in it.
